pxl_scripts/px/perf_flamegraph/vis.json

```diff
@@ -0,0 +1,82 @@
+{
+  "variables": [
+    {
+      "name": "start_time",
+      "type": "PX_STRING",
+      "description": "The relative start time of the window. Current time is assumed to be now",
+      "defaultValue": "-5m"
+    },
+    {
+      "name": "node",
+      "type": "PX_NODE",
+      "description": "The node to filter on.",
+      "defaultValue": ""
+    },
+    {
+      "name": "namespace",
```

Comment:
I'd switch the order of the arguments: namespace > node > pod makes more sense.

Reply:
Acutally, node is independent of namespace/pod/container/pid.

For example, a node can contain multiple namespaces. Also, a namespace can span multiple nodes. So it's really an any-to-any relationship, not a parent-child relationship.

So we need to keep the namespace and pod filters adjacent to each other. Those two do have a parent-child relationship.

Reply:
Oh, whoops. I'm not sure what I was thinking. :)